docs/performance.md

```diff
@@ -90,12 +90,15 @@ To reproduce, run `sbt bench` (this will take a while).
 |--------------------|----------------------|----------------------|
 | RscNativeTypecheck | 300.146 ms           | 282.468 ms           |
 | RscTypecheck       | 476.843 ± 0.822 ms   | 33.685 ± 0.024 ms    |
+| ScalacNamer 2.12.4 | 1843.751 ± 14.930 ms | 29.825 ± 0.090 ms    |
+| ScalacNamer 2.11.11| 1423.331 ± 37.056 ms | 71.100 ± 0.318 ms    |
 | ScalacTypecheck    | 4326.812 ± 28.763 ms | 712.872 ± 3.554 ms   |
 | ScalacCompile      | 8098.704 ± 47.390 ms | 1691.732 ± 12.205 ms |
 | JavacCompile       | 851.787 ± 3.460 ms   | 76.164 ± 0.169 ms    |
 
 ## Comments
-
+  * Compared to the Scala namer in Scala 2.11.11, our prototype type-checker is about 2x faster.
+    However, compared to the namer in 2.12.4 we are about 10% slower.
```

Comment:
As @retronym, the member of the Lightbend Scala compiler team, alluded to in the comments above, the Rsc typecheck phase does significantly more work than the Scalac namer phase. 

The namer phase in Scalac only traverses packages and top-level definitions from the sources, creating symbols for them without performing name resolution. The typecheck phase in Rsc fully populates the symbol table from the sources and resolves all names - starting with signatures and ending with identifiers in method bodies.

The closest phase in Rsc is schedule, which traverses all global definitions (definitions that can be visible from other files), creating an analogue of symbols for them, also without performing name resolution. Please update your benchmark results and this comment.

Here are the preliminary results from a quick run on my laptop. Even though Rsc schedule does significantly more work, it is still 3x faster than Scalac's namer.
```
  |--------------------|----------------------|----------------------|
  | RscSchedule        | 407.193 ± 3.582      | 9.844 ± 0.078 ms     |      
  | ScalacName 2.12.4  | 1843.751 ± 14.930 ms | 29.825 ± 0.090 ms    |
  | ScalacName 2.11.11 | 1423.331 ± 37.056 ms | 71.100 ± 0.318 ms    |
  | RscTypecheck       | 476.843 ± 0.822 ms   | 33.685 ± 0.024 ms    |
  | ScalacTypecheck    | 4326.812 ± 28.763 ms | 712.872 ± 3.554 ms   |
```